Sources/Common/DataModel/Spline1D/index.js

```diff
@@ -22,7 +22,16 @@ function vtkSpline1D(publicAPI, model) {
 
   // --------------------------------------------------------------------------
 
-  publicAPI.computeOpenCoefficients = (size, work, x, y) => {
+  publicAPI.computeOpenCoefficients = (
+    size,
+    work,
+    x,
+    y,
+    leftConstraint,
+    leftValue,
+    rightConstraint,
+    rightValue
```

Comment:
leftConstraint, leftValue, rightConstraint and rightValue should probably be moved into an "option" object, e.g.
`publicAPI.computeOpenCoefficients(size, work, x, y, {leftConstraint, leftValue, rightConstraint, rightValue})` where `publicAPI.computeOpenCoefficients = (size, work, x, y, options = {})`